osh/cmd_eval.py

```diff
@@ -437,11 +437,12 @@ def _EvalRedirect(self, r):
                     # with globs
 
                     # mycpp needs this explicit declaration
-                    b = braces.BraceDetect(arg_word)  # type: Optional[word.BracedTree]
+                    b = braces.BraceDetect(
+                        arg_word)  # type: Optional[word.BracedTree]
```

Comment:
Hm I'm getting different formatting

Can you try running

```
devtools/format.sh install-yapf
```

Then

```
$ devtools/format.sh yapf-version 
yapf 0.40.1
```
devtools/format.sh yapf-files
```

It should print version 0.40.1

And then `devtools/format.sh yapf-files osh/cmd_eval.py`

Reply:
Hmmm. My `yapf` version is also `0.40.1`. And, even after reinstalling, I cannot get the same result as you.

FWIW I ran:

```sh
$ devtools/format.sh install-yapf
$ devtools/format.sh yapf-version
$ devtools/format.sh yapf-files osh/cmd_eval.py
```

I am honestly not sure what the issue is then. Do you have any ideas?

In case it helps, here's what I get when manually checking the yapf version:

```sh
$ source _tmp/yapf-venv/bin/activate
(yapf-venv) $ pip list
Package            Version
------------------ -------
importlib-metadata 6.8.0
pip                22.0.4
platformdirs       3.10.0
setuptools         58.1.0
tomli              2.0.1
yapf               0.40.1
zipp               3.17.0
WARNING: You are using pip version 22.0.4; however, version 23.2.1 is available.
You should consider upgrading via the '/home/aolse/oils/oil/_tmp/yapf-venv/bin/python3 -m pip install --upgrade pip' command.
```

And even manually invoking this version of yapf:

```sh
(yapf-venv) $ python3 -m yapf -i osh/cmd_eval.py
(yapf-venv) $ git status
On branch argparse
Your branch is up to date with 'origin/argparse'.

nothing to commit, working tree clean
```

Reply:
Oh I should have said that I do get some diffs in `osh/cmd_eval.py`, but not the same ones as in this PR

Maybe we can debug it on Zulip a little ...

It would be nice to get the same diffs, then possibly merge the formatting change separate from the  other changes, just so it's easier to see what happened

And smalller diffs tend to make merges easier

But it's not essential, we can play with it a little

Reply:
I'll send a PR to show what I mean, maybe we can get to the bottom of this